### PR TITLE
[dom4] Initial commit.

### PR DESCRIPTION
**Pull request by jeluard**

### CODE REVIEW

File: dom4/build.boot [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+    (download :url (str "https://cdnjs.cloudflare.com/ajax/libs/dom4/" +lib-version+ "/dom4.js"))
+    (download :url (str "https://cdnjs.cloudflare.com/ajax/libs/dom4/" +lib-version+ "/dom4.max.js"))
+    (sift :move {#"dom4.max.js" "cljsjs/development/dom4.inc.js"
+                 #"dom4.js" "cljsjs/production/dom4.min.inc.js"})
+    (sift :include #{#"^cljsjs"})
+    (deps-cljs :no-externs true :name "cljsjs.dom4")))
```
Comment (martinklepsch) [inline dom4/build.boot]: why does this have `:no-externs true` but still provides an externs file?

Comment (jeluard) [inline dom4/build.boot]: My bad this should not be there.

**Comment by martinklepsch** [comment]
Looking good!

**Comment by martinklepsch** [comment]
Thanks Julien, just deployed to Clojars :)

**Comment by jeluard** [comment]
Thanks! I am on the road right now so couldn't easily do it myself.

**Comment by martinklepsch** [comment]
@jeluard no worries :)

**Pull request closed without merging**